target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -384,6 +384,11 @@ tiny-ac)
 	set_wifi_led "$boardname:orange:wifi"
 	set_usb_led "$boardname:green:usb"
 	;;
+tl-wa801nd-v5)
+	set_wifi_led "$boardname:green:wlan"
```

Comment:
As this device is using mt76 wifi driver, you should consider changing the LED trigger as described here: https://github.com/openwrt/openwrt/commit/82fa7b9d78acf53708ee6480df328b11023e4a56